.nix/config.nix

```diff
@@ -15,7 +15,7 @@ with builtins; with (import <nixpkgs> {}).lib;
 
   ## Indicate the relative location of your _CoqProject
   ## If not specified, it defaults to "_CoqProject"
-  coqproject = "mathcomp/_CoqProject";
+  coqproject = "_CoqProject";
```

Comment:
This does not belong to this PR.

Reply:
Sure, but it is its own trivial commit. Probably not worth its own PR if you plan to merge this relatively rapidely?